modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/pageValidationReducer.es.js

```diff
@@ -63,7 +65,14 @@ export default (state, action) => {
 			);
 
 			if (firstInvalidFieldInput) {
-				firstInvalidFieldInput.focus();
+				if (firstInvalidFieldInput.type !== 'hidden') {
```

Comment:
Great catch, @joaovictoralvesmartins! :tada: :clap:

Reply:
@natocesarrego, haha! Thanks! I've noticed that the form builder uses hidden input fields to store the values, so i thought that was a good place to start :D